tests/attr/test_lrp.py

```diff
@@ -124,19 +124,6 @@ def test_lrp_simple_repeat_attributions(self) -> None:
         output_after = model(inputs)
         assertTensorAlmostEqual(self, output, output_after)
 
-    def test_lrp_simple_inplaceReLU(self) -> None:
```

Comment:
@vivekmig, there are several pre-trained models that have `inplace=True` for ReLU. I think @nanohanno, specifically needed it for his models, if I remember correctly. Does this mean that we need to make that change in the pre-trained models ?

Reply:
Yes, I think we would need to change this for pre-trained models as well. I didn't see any explicit documentation of this limitation, but obtained an error with toy models using in-place modules, as below. Maybe @albanD can provide more context or documentation?

```
import torch
import torch.nn as nn

class SampleRelu(nn.Module):
    def __init__(self):
        super().__init__()
        self.relu = nn.ReLU(inplace=True)

    def forward(self, inp):
        return self.relu(inp)
relu_mod = SampleRelu()

def grad_hook(mod, inp, out):
  print("HOOK")
hook = relu_mod.relu.register_full_backward_hook(grad_hook)
inp = torch.randn(3,4, requires_grad=True)
out = relu_mod(inp)
```
Result:
```
RuntimeError: Output 0 of BackwardHookFunctionBackward is a view and is being modified inplace. This view was created inside a custom Function (or because an input was returned as-is) and the autograd logic to handle view+inplace would override the custom backward associated with the custom Function, leading to incorrect gradients. This behavior is forbidden. You can fix this by cloning the output of the custom Function.
```

Reply:
Hey!
The reason is that for inplace ops, the input and output to your nn.Module is effectively the same Tensor. So it is not clear which gradient you expect to see there: the same or two different things?

The other thing is that for Module with multiple inputs or outputs, we can't technically do it. We can special case the one input/output case though if you think the input's gradient should be for the value before it was modified and the output's for the value after it was modified?

Reply:
Ok, I had this exact same issue half an hour ago.... as a dirty fix to my model, I just changed all my inplace Relu to not in place ones:
```python
def correct_inplace_relu(model):
    for child_name, child in model.named_children():
        if isinstance(child, nn.ReLU):
            setattr(model, child_name, nn.ReLU())
        else:
            correct_inplace_relu(child)
correct_inplace_relu(model)
```
Afterwards, everything works fine.